.github/workflows/create_llvm.yml

```diff
@@ -15,18 +18,33 @@ jobs:
         version: [17, 18]
         os: [ubuntu-22.04]
         build_type: [Release, RelAssert]
+        arch : [x86_64, aarch64]
+        exclude:
+          # For now just do latest llvm with aarch to reduce cache usage.
+          - arch: aarch64
+            version: 17
+          - arch: aarch64
+            build_type: Release
         include:
           # We want to set flags related to particular matrix dimensions. To do this
           # we need to create default values first, and then against particular matrix
           # dimensions.
           # Note that we need to use RelAssert as the cache key matching can match Release against ReleaseAssert
           - os_flags:
+          - arch_flags:
           - build_type_flags:
-          - build_type: RelAssert
-            build_type_flags: -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_ASSERTIONS=ON
+          - build_type_flags: -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_ASSERTIONS=ON
+            build_type: RelAssert
+          - arch_flags: -DCMAKE_TOOLCHAIN_FILE="$GITHUB_WORKSPACE/ock/platform/arm-linux/aarch64-toolchain.cmake"
+                        -DLLVM_TARGET_ARCH=AArch64
```

Comment:
`LLVM_TARGET_ARCH` should not be necessary, that's for when you build LLVM *as* a cross-compiler, rather than when you build LLVM *using* a cross-compiler.

Reply:
I can drop it, but it's what we have in our documentation.

Reply:
Yeah, there's a bunch of things in the documentation and in our internal GitLab builds that are not necessary but also not harmful. I've just ignored them but if we're moving things over to GitHub now seems like a good time to clean that up.